controllers/configurationpolicy_controller.go

```diff
@@ -2393,8 +2393,9 @@ func (r *ConfigurationPolicyReconciler) handleSingleObj(
 
 		created := false
 		uid := string(obj.existingObj.GetUID())
+		dryRunNoOpOverride := false
 
-		if evaluated, compliant, cachedMsg := r.alreadyEvaluated(obj.policy, obj.existingObj); evaluated {
+		if evaluated, compliant, cachedMsg, dryRunNoOp := r.alreadyEvaluated(obj.policy, obj.existingObj); evaluated {
```

Comment:
Is it necessary to store `dryRunNoOp` in the cache of already evaluated things? It seems like it could be pulled from the status, like `Properties.Diff` is.

Reply:
Good point, I can remove it from the cache